extensions/oni-plugin-markdown-preview/src/index.tsx

```diff
@@ -168,11 +171,30 @@ class MarkdownPreview extends React.PureComponent<IMarkdownPreviewProps, IMarkdo
 
 class MarkdownPreviewEditor implements Oni.IWindowSplit {
     private _open: boolean = false
+    private _unrenderedContent: string = ""
+    private _renderedContent: string = ""
 
     constructor(private _oni: Oni.Plugin.Api) {
         this._oni.editors.activeEditor.onBufferEnter.subscribe(args => this.onBufferEnter(args))
     }
 
+    public isPaneOpen(): boolean {
+        return this._open
+    }
+
+    public getUnrenderContent(): string {
```

Comment:
Minor nitpick - should this be `unrenderedContent`?

Reply:
Oops...